packages/app/helper/getDays.js

```diff
@@ -0,0 +1,16 @@
+export function getDays(timeStamp) {
+  let daysPassed
+  const days = Math.ceil(
+    (new Date() - new Date(timeStamp.seconds * 1000)) / (1000 * 3600 * 24)
+  ) -1
```

Comment:
Maybe I'm mistaken but couldn't you use Math.floor here instead of doing a Math.ceil and then subtracting the result?